packages/cli/src/cli.ts

```diff
@@ -70,6 +70,16 @@ export async function cli(args: Array<string>, cli: CliOptions): Promise<void> {
     await printUsage(stdout, env);
     throw null;
   }
+
+  // Add optional typescript support by loading tsx or ts-node
```

Comment:
Could you give a bit of context as to what this is doing?

Reply:
yes - there is a problem, when running the tests for the whole CLI (i call them e2e cli tests) vitest typescript transpiler will not work because there is dynamic module loading and worker threads.
So, to make it work, we need to load "tsx"

On the other end, this will also give the possibility to users to load a ts configuration file - that is not important but comes for free.

The idea is that we load tsx or ts-node transpilers if they are installed in node_modules

I really needed a cli test to have something consistent to run to try things out, and, we want to keep it for future regressions for sure.